### DISCUSSION TITLE
Generate a population of 100 but got 124 records in Patients

### DISCUSSION THREAD

**Discussion by aiya6502** [synthetichealth/synthea#1195] [General]
Any clue that how come I got 124 records in patients.csv file after I issued the following command ?
java -jar synthea-with-dependencies.jar -c /{PATH_TO synthea.properties file} -p 100 

My JDK version is 11.0.16.1 running on Windows 11 and the Database in using is PostgreSQL 15.0 , and in the synthea.properties:
1. I copied the file from sample one
2. Update "exporter.fhir.export = false" and "exporter.csv.export = true"

Thanks.

**Comment by jawalonoski** [comment]
The `-p` switch is a shortcut for `generate.default_population`: 
> This is the number of living patients (population size) that Synthea will generate. Synthea will generate patients until the number of living patients reaches this value.

From: See https://github.com/synthetichealth/synthea/wiki/Common-Configuration#generation-settings.

If you have a town with a census population of 100 people, that means 100 living people. In the simulation, sometimes people die. So, they don't count in terms of total population.

If you don't like how that works, you can modify the following settings in `synthea.properties`:

```properties
# Set to true if you want every patient to be dead.
generate.only_dead_patients = false
# Set to true if you want every patient to be alive.
generate.only_alive_patients = false
# If both only_dead_patients and only_alive_patients are set to true,
# It they will both default back to false
```

**Reply by aiya6502** [reply]
Got it, thanks a lot for clarifying this.

**Reply by shoshinL** [reply]
Thank you as well. I was looking for this information for quite a while, since I overread the "living" in the wiki.
Would setting one of these options to true cause health outcomes that cause patients to die to not be modeled?

**Reply by jawalonoski** [reply]
> Thank you as well. I was looking for this information for quite a while, since I overread the "living" in the wiki. Would setting one of these options to true cause health outcomes that cause patients to die to not be modeled?

No, the health outcomes are still modeled. If you are forcing patients to only be alive, the simulation will discard any patient that died during the simulation and try again until it gets one who survives long enough.
